addons/l10n_id_efaktur_coretax/models/res_partner.py

```diff
@@ -20,3 +19,28 @@ class Partner(models.Model):
         ('Other', 'Others')
     ], default='TIN', string="Document Type")
     l10n_id_buyer_document_number = fields.Char(string="Document Number")
+    l10n_id_nik = fields.Char(string='NIK')
+
```

Comment:
Try not to add random blank lines; if you want to separate your fields a lien with a  comment is better IMO